pyproject.toml

```diff
@@ -130,7 +130,7 @@ exclude = ["doc/", "pytensor/_version.py"]
 docstring-code-format = true
 
 [tool.ruff.lint]
-select = ["B905", "C", "E", "F", "I", "UP", "W", "RUF", "PERF", "PTH", "ISC", "T20", "NPY201"]
```

Comment:
I removed B905 (zip explicit strict) because passing `strict=False` is actually slower by 100ns than `strict=None` which adds up in hot loops. Furthermore the `noqa: B905` didn't really work in nested comprehensions, ruff would remove it as useless and then complain the strict kwarg was missing.

Reply:
Ricardo v Dev Tools, season 2